source/fundamentals/enterprise-authentication.txt

```diff
@@ -346,7 +350,11 @@ see the corresponding syntax.
       The following code example shows how to specify GCP IMDS OIDC authentication as
       part of the authentication string. 
       Replace the ``<audience>`` placeholder with the
-      value of the ``audience`` parameter configured on your MongoDB deployment. 
+      value of the ``audience`` parameter configured on your MongoDB deployment.
+
+      You can't pass values containing the comma (``,``) character to the ``authMechanismProperties`` 
+      option. You must specify values that contain commas in a ``MongoCredential`` object, 
+      as demonstrated in the :guilabel:`MongoCredential` tab.
```

Comment:
S: I wonder if these should be in `.. note::` admonitions, since I would assume this is a smaller edge-case and it seems to be a sort of side-comment before the example. That being said, part of me is also in favor of less admonitions, so I can be swayed either way. I'll leave it to your discretion, just wanted to note my thoughts on it!

Reply:
Ah I see what you mean. In the ticket, there was a link to the [Java docs](https://www.mongodb.com/docs/drivers/java/sync/current/fundamentals/enterprise-auth/#mongodb-oidc) as an example that have a similar caveat in the description, so I followed that pattern here instead of following the `.. note::` route.